apps/api/e2e/utils.ts

```diff
@@ -0,0 +1,122 @@
+//# cSpell:disable
```

Comment:
Is there a particular reason we moved away from [fakerjs](https://fakerjs.dev/)?

Reply:
[REF ](https://novu.slack.com/archives/C06DU7A7BPV/p1717411734557139)

Reply:
thank you :)